### ISSUE TITLE
1/3rd caster pints calculating wrong

### ISSUE THREAD

**Issue by Battenhatch** [Mary-Usagi/spellpoints5e#6]
Hi

I've come across an issue. I'm GMing a 7th level Arcane Trickster, and it calculates to 6 spell point. A 7th level Arcane Trickster has 4 1st level and 2 2nd level slots normally, which normally turns into 14 spell points (4 x 2 +  2 x 3). Basically, 1/3rd level casters divide their level by 3, rounding UP, to determine their number of slots when single classed, but that's sometimes a bit confused by them adding 1/3rd their level, rounded DOWN, when multiclassing. I don't know if that's what's happening here.

Regards,

Claus Bundgaard Rasmussen

**Comment by DaviCaamano** [comment]
Have this exact same problem. It rounds down instead of up like in the DMG. This is a problem for half casters too. 

You need to add levels in the class to fix it for now without increasing any features or health points.

**Comment by Battenhatch** [comment]
Yes, just found out about the half-casters myself today ;-). Let's hope someone can rectify :)

**Comment by Battenhatch** [comment]
Actually, I just went back and checked the rules. Then I checked Sage Advice. Under the Spell Point system, this is the intension, actually. The module is right. See this: https://www.sageadvice.eu/spell-points-variant/

This being the case, I'd like to change this to the suggestion that the effective caster level can be based on the "normal" progression, ie. rounding UP for single class half and third casters.
